### ISSUE TITLE
No error when stubs are invalid

### ISSUE THREAD

**Issue by xenoterracide** [typetools/checker-framework#4586]
I've been creating some stubs, mostly invalid the first time around, but there is no error telling me this. I've done things as obvious as forgetting the return type

```java
@Nullable getParent();
```

or as non obvious as excluding final on a class

```java
public class Objects {
```

but checker didn't tell me this, which makes it harder to tell why an error isn't caught, and also makes it more likely that an error isn't caught at all because you don't realize that your checks are broken.

**Comment by mernst** [comment]
Please use the `-AstubWarnIfNotFound` command-line option.
It is described in manual section ["Troubleshooting stub libraries"](https://checkerframework.org/manual/#stub-troubleshooting).

I feel your pain about the dumb typos -- I have done that a lot too.

These warnings are not enabled by default because they would produce too much output.  An example of an innocuous warning is when a stub file is written for library version B but an earlier version of the library is on the classpath; a warning is issued for every method in the stub file but not in the earlier version of the library.  (This happens a lot for the JDK, and also for other libraries.)

Perhaps the warnings could be enabled by default for any stub file provided via the `-Astubs` command-line option, but not stub files built into the checker.

**Comment by xenoterracide** [comment]
it looks like it is throwing warnings, but it might be nicer if it threw errors if the files are not parseable.
